src/python_testing/TC_DeviceConformance.py

```diff
@@ -76,17 +76,23 @@ def record_warning(location, problem):
 
         success = True
         allow_provisional = self.user_params.get("allow_provisional", False)
+        # TODO: automate this once https://github.com/csa-data-model/projects/issues/454 is done.
+        provisional_cluster_ids = [Clusters.ContentControl.id, Clusters.ScenesManagement.id]
```

Comment:
sure. When I originally was doing this, I didn't think it would be a problem because those other clusters didn't have any backing implementation in the TH. Let me just check and see what landed in the sve branch and I'll add the provisional ones that can be added.

Reply:
added. Checked on the 1.3-sve branch and that also looks OK.